internal/rm/device_map.go

```diff
@@ -37,9 +37,13 @@ type DeviceMap map[spec.ResourceName]Devices
 
 // NewDeviceMap creates a device map for the specified NVML library and config.
 func NewDeviceMap(nvmllib nvml.Interface, config *spec.Config) (DeviceMap, error) {
+	migStrategy := spec.MigStrategyNone
+	if config.Flags.MigStrategy != nil {
+		migStrategy = *config.Flags.MigStrategy
+	}
```

Comment:
Does it make sense to do this at a higher level? Possibly even in the config parser?

Reply:
Please see the follow-up commit to add the default to the CLI. I can create a follow up to set the default in the config parser instead (after dropping this commit).

Reply:
I have now set the default in the CLI.